postman.json

```diff
@@ -226,11 +241,12 @@
 									"    pm.expect(pm.response.code).to.equal(404);\r",
 									"});\r",
 									"/* {\r",
-									"    \"message\": \"Requested UploadSlot couldn't be found.\"\r",
+									"    \"message\": \"No query results for model [App\\\\Models\\\\UploadSlot] wrongImageToken\"\r",
```

Comment:
This message changed as Laravel changed the way the exception handler are registered: https://laravel.com/docs/11.x/errors

This was previously managed in `app/Exceptions/Handler`

Reply:
We might want to re-add this.

Reply:
done